android/build.gradle

```diff
@@ -31,7 +31,7 @@ android {
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
   buildToolsVersion getExtOrDefault('buildToolsVersion')
   defaultConfig {
-    minSdkVersion 21
+    minSdkVersion 24
```

Comment:
This solved the following error:
```
Execution failed for task ':app:mergeExtDexDebug'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform rive-android-9.2.2.aar (app.rive:rive-android:9.2.2) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=21, org.gradle.category=library, org.gradle.status=release, org.gradle.usage=java-runtime, org.jetbrains.kotlin.platform.type=androidJvm}.
      > Execution failed for DexingWithClasspathTransform: /Users/philrive/.gradle/caches/transforms-3/defcb3a484b64167fc84e6cd1cef719b/transformed/jetified-rive-android-9.2.2-runtime.jar.
         > Error while dexing.
   > Failed to transform kotlin-stdlib-1.9.22.jar (org.jetbrains.kotlin:kotlin-stdlib:1.9.22) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=21, org.gradle.category=library, org.gradle.jvm.environment=standard-jvm, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime, org.jetbrains.kotlin.platform.type=jvm}.
      > Execution failed for DexingWithClasspathTransform: /Users/philrive/.gradle/caches/transforms-3/755bc55d71b5f70c0d2fe3d2d5f63094/transformed/jetified-kotlin-stdlib-1.9.22.jar.
         > Error while dexing.
```

Reply:
I believe we don't need to update the minSDKVersion here, and it'll only be needed in the example/build.gradle to get the example compiling.

I also suspect that there is just some other config we need to update in the example project, instead of bumping the minSDK.